src/delegate.erl

```diff
@@ -200,6 +235,16 @@ group_pids_by_node(Pids) ->
                  node(Pid), fun (List) -> [Pid | List] end, [Pid], Remote)}
       end, {[], maps:new()}, Pids).
 
+group_local_call_pids_by_node(Pids) ->
+    {LocalPids0, Grouped0} = group_pids_by_node(Pids),
+    maps:fold(fun(K, V, {AccIn, MapsIn}) -> 
+        case V of
+            %% just one Pid for the node
+            [S_v] -> {[S_v | AccIn], MapsIn};
```

Comment:
`S_v` uses an unorthodox naming of bindings (variables) which you won't find used elsewhere in RabbitMQ.

Reply:
S_v  is Single_value

Reply:
"erlang:raise/3" is used because it is copied from the original function "invoke(Pid, Name, FunOrMFA) when is_pid(Pid)"